nominees/operation-serenata-de-amor.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Operation Serenata de Amor",
+  "aliases": [
+    ""
+  ],
+  "description": "An open project that uses data science - the same technologies used by giants like Google, Facebook and Netflix - for the purpose of monitoring public spending and sharing information in a way accessible to everyone. Serenata has created Rosie: an artificial intelligence who analyzes Brazilian congresspeople expenses while they are in office. Rosie can find suspicious spendings and engage citizens in the discussion about these findings.",
```

Comment:
Is this submission for Serenata or Rosie? The terms and names keep getting mixed up. I'm not clear about what this nomination should focus on.

Reply:
https://github.com/okfn-brasil/serenata-de-amor#how-1